SudoScript.Core/Solver.cs

```diff
@@ -76,9 +76,17 @@ public static Board GenerateSolveable(Board board)
         throw new NotImplementedException();
     }
 
-    public static bool IsSatisfactory(Board board)
+    /// <summary>
+    /// Checks if the board can be solved by just using the EliminateCandidates methods from units.
+    /// </summary>
+    /// <param name="board"></param>
+    /// <returns>True if the board can be solved without trial and error guessing.</returns>
+    public static bool IsSatisfactory(Board board) // Certain methods for eliminating candidates using inference are not currently implemented. Implementing them would make this function more acurate.
```

Comment:
Commented on this already in #79

Reply:
Satisfactory means you can get the solutions without guess-work. Proper is when there is only one solution.

Reply:
As we discussed in the report both of those actually mean the same thing. If you dont need guess work you can always arrive at the single solution. The only time you would ever need guess work is if there is more than one solution.

Reply:
A sudoku with a single solution does not have to be satisfactory. It depends how you define satisfactory. How much inference is enough to make it unsatisfactory? I feel like using this method as a qualifier for satisfaction is fine, since the IsProper method is separate anyways.